stable/rabbitmq/README.md

```diff
@@ -119,6 +119,13 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `metrics.image.repository`           | Exporter image name                              | `kbudde/rabbitmq-exporter`                              |
 | `metrics.image.tag`                  | Exporter image tag                               | `v0.29.0`                                               |
 | `metrics.image.pullPolicy`           | Exporter image pull policy                       | `IfNotPresent`                                          |
+| `metrics.serviceMonitor.enabled`     | Create ServiceMonitor Resource for scraping metrics using PrometheusOperator   | `false`                   |
+| `metrics.serviceMonitor.namespace`   | Namespace where servicemonitor resource should be created         | `the same namespace as nginx ingress`  |
```

Comment:
`the same namespace as nginx ingress` is this default value correct? In the values files I see the namespace not set so you should use `nil`

Reply:
Sorry, distraction error. You're absolutely right. I fixed it.